boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -32,7 +32,9 @@ endif
 CARGO_FLAGS += -Zpanic-abort-tests
 
 .PHONY: ot-check
-OPENTITAN_ACTUAL_SHA := $(shell cd $(OPENTITAN_TREE); git show --pretty=format:"%H" --no-patch)
+ifeq ($(OPENTITAN_TREE),)
```

Comment:
This is suppsoed to be ifneq, somehow I messed things up rebasing.

Reply:
I think this is caught by the below test for `OPENTITAN_ACTUAL_SHA == OPENTITAN_SUPPORTED_SHA` (which will be false when empty, and can be overridden by `SKIP_OT_VERSION_CHECK`). The `OPENTITAN_ACTUAL_SHA` seem to be really only used for that.

Reply:
> I think this is caught by the below test for `OPENTITAN_ACTUAL_SHA == OPENTITAN_SUPPORTED_SHA` (which will be false when empty, and can be overridden by `SKIP_OT_VERSION_CHECK`). The `OPENTITAN_ACTUAL_SHA` seem to be really only used for that.

I guess it's probably worth cleaning up that logic as well.

Reply:
Turns out there's not much to cleanup. We need to set the `OPENTITAN_ACTUAL_SHA` variable before the ot-check target because you can't mix variable assignment with recipes.